tripy/docs/pre0_user_guides/00-introduction-to-tripy.md

```diff
@@ -90,8 +90,6 @@ end = time.time()
 
 compiled_time = (end - start) * 1000
 print(f"Compiled mode time: {compiled_time:.4f} ms")
-# Make sure compiled mode is actually faster # doc: omit
-assert compiled_time < 0.01 * eager_time # doc: omit
```

Comment:
this was the failure at the doc, the previous `mlp` call gets cached so the call to eager mode `mlp` in this test is pretty fast.  I removed this assert for now

Reply:
Can we clear the cache in between? You can add a commented out block like:
```md
<!--
```py
... # Clear cache here
```
-->

Reply:
that worked!